src/components/MarketplaceCard.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <BaseCard :title="title" class="marketplace">
+    <div class="content shadow">
+      <p>{{ title }}</p>
+      <p>by: {{ author }}</p>
+      <p>{{ rating }}</p>
+      <p>{{ tags }}</p>
+    </div>
+  </BaseCard>
+</template>
+<script>
+import BaseCard from '@/components/BaseCard'
+export default {
+  name: 'MarketplaceCard',
+  props: {
+    title: String,
+    text: String,
+    author: String,
+    rating: Number,
+    tags: String,
+    cover: String,
+  },
+  components: {
+    BaseCard,
+  },
+  methods: {
+    handleKeyUp: (event) => {
+      if (event.keyCode === 13) {
+        this.props.nextStep()
+      }
+    },
+  },
+}
+
+</script>
+<style lang="scss">
+  @import '@/styles/app.global.scss';
+  @import '@/styles/_colors.scss';
+
+  .marketplace>.title {
```

Comment:
Add spaces between selectors
```suggestion
  .marketplace > .title {
```